1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+---
+post_title: Quick Start
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+
+# Launch a long running interactive Bash session
+
+In this example, a long running job app is launched and then the `dcos exec` command is used to launch an interactive Bash session inside of the task container on the node.
```

Comment:
I wouldn't call it a `job app` I would just call it a `job` and maybe link to any metronome docs you have.

Additionally, I would probably reword this whole sentence as:
```
In this example, a long running job is launched via the `dcos job run` command and we use `dcos task exec` to launch an interactive Bash shell inside that job's container.
```